src/components/widgets/MetadataWidget/BreadcrumbWidget/BreadcrumbWidget.tsx

```diff
@@ -31,54 +32,36 @@ export interface BreadcrumbWidgetProps {
 
 const NO_SHORTFORM = "No short form available.";
 
-async function getShortForm(olsApi: OlsApi, entityType: string, ontologyId?: string, iri?: string, parameter?: string): Promise<string> {
-  if (entityType == "term"){
-    const response = await olsApi.getTerm(undefined, undefined, {ontologyId: ontologyId, termIri: iri}, parameter)
-        .catch((error) => console.log(error));
-    if (response?._embedded?.terms[0].short_form != null && response._embedded.terms[0].short_form != null) {
-      return response._embedded.terms[0].short_form.toUpperCase();
-    } else {
-      return NO_SHORTFORM;
-    }
-  }
-  if (entityType == "property"){
-    const response = await olsApi.getProperty(undefined, undefined, {ontologyId: ontologyId, propertyIri: iri}, parameter)
-        .catch((error) => console.log(error));
-    if (response?._embedded?.properties[0].short_form != null && response._embedded.properties[0].short_form != null) {
-      return response._embedded.properties[0].short_form;
-    } else {
-      return NO_SHORTFORM;
-    }
-  }
-  if (entityType == "individual"){
-    const response = await olsApi.getIndividual(undefined, undefined, {ontologyId: ontologyId, individualIri: iri}, parameter)
-        .catch((error) => console.log(error));
-    if (response?._embedded?.individuals[0].short_form != null && response._embedded.individuals[0].short_form != null) {
-      return response._embedded.individuals[0].short_form;
-    } else {
-      return NO_SHORTFORM;
-    }
-  }
-  //unacceptable object type
-  return NO_SHORTFORM;
-}
-
 function BreadcrumbWidget(props: BreadcrumbWidgetProps) {
   const { api, ontologyId, iri, entityType, colorFirst, colorSecond, parameter } = props;
   const fixedEntityType = entityType == "class" ? "term" : entityType
   const olsApi = new OlsApi(api);
 
   const {
-    data: shortForm,
-    isLoading,
-  } = useQuery([api, "short_form", fixedEntityType, ontologyId, iri, parameter], () => { return getShortForm(olsApi, fixedEntityType, ontologyId, iri, parameter); });
+    data: ontologyJSON,
+    isLoading: isLoading,
+    isSuccess: isSuccess,
+  } = useQuery([api, "short_form", fixedEntityType, ontologyId, iri, parameter], () => { return getPreferredOntologyJSON(olsApi, fixedEntityType, ontologyId, iri, parameter); });
 
   return (
-      <span>
-        <EuiBadge color={colorFirst || "primary"}>{ontologyId?.toUpperCase()}</EuiBadge>
-        {" > "}
-        <EuiBadge color={colorSecond || "success"}>{shortForm}</EuiBadge>
-      </span>
+      <>
+        {isLoading && <EuiLoadingSpinner size="s"></EuiLoadingSpinner>}
+        {isSuccess &&
+            <span>
+              {
+                  !props.ontologyId && !ontologyJSON["is_defining_ontology"] &&
+                  <EuiFlexItem>
+                    <EuiText>
+                      <i>Defining ontology not available. Showing occurrence inside {ontologyJSON["ontology_name"]} instead.</i>
```

Comment:
The info message should not change the shape of the BreadcrumbWidget so much. The large info message may lead to unwanted UI changes. Maybe you can adapt the text size to the text size of the text in the badges and add the second part (Showing occurrence inside ontology instead) as EUI Tooltip.

Reply:
Looks like this now:
![grafik](https://github.com/nfdi4health/semlookp-widgets/assets/141128943/3e832925-7e4d-4806-bdc2-cd1347d4088e)

Should this look maybe get applied to all widgets? I think it's not only more convenient for the Breadcrumb widget